### PR TITLE
BC-5583 - Display pdf preview

### PR DESCRIPTION
**Pull request by bischofmax**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/feature-board-file-element/content/display/FileDisplay.vue [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <template>
 	<div>
+		<PdfDisplay
+			v-if="hasPdfMimeType && fileProperties.previewUrl"
+			:src="fileProperties.url"
+			:preview-src="fileProperties.previewUrl"
+			:name="fileProperties.name"
+			:element="fileProperties.element"
+		/>
```
Comment (SevenWaysDP) [inline src/components/feature-board-file-element/content/display/FileDisplay.vue:9]: Hier vllt Gedankenanstoß:
Warum übergeben wir nicht fileProperties ins jeden DisplayComponent und er entscheidet ob er erscheint?

File: src/components/feature-board-file-element/content/display/image-display/ImageDisplay.unit.ts [hunk 2]
```diff
@@ -79,84 +79,20 @@ describe("ImageDisplay", () => {
...
-
-		it("should display loading spinner", () => {
-			const { wrapper } = setup({ isEditMode: false });
-
-			const loadingSpinner = wrapper.findComponent({
-				name: "VProgressCircular",
-			});
-
-			expect(loadingSpinner.exists()).toBe(true);
-		});
```
Comment (bergatco) [inline src/components/feature-board-file-element/content/display/image-display/ImageDisplay.unit.ts:120]: Why did you remove the tests for the spinner? Is it no longer testable, as it uses now basic functionalities of `v-img`?

**Review by bergatco — APPROVED** [review]

File: src/components/feature-board-file-element/content/display/FileDisplay.unit.ts [hunk 3]
```diff
@@ -321,9 +391,17 @@ describe("FileDisplay", () => {
...
+				const { wrapper } = setup();
+
+				const pdfDisplay = wrapper.findComponent(PdfDisplay);
+
+				expect(pdfDisplay.exists()).toBe(false);
+			});
 		});
 
-		describe("when mimeType is not a video or audio or pdf type", () => {
+		describe("when mimeType is not a video or audio type", () => {
```
[the remark this answers is not in the document]
Comment (SevenWaysDP) [reply src/components/feature-board-file-element/content/display/FileDisplay.unit.ts:404]: Because it now has its own component

File: src/components/feature-board-file-element/content/display/image-display/ImageDisplay.unit.ts (hunk 2 above)
Comment (SevenWaysDP) [reply src/components/feature-board-file-element/content/display/image-display/ImageDisplay.unit.ts:120]: I have created it now

**Review by bergatco — APPROVED** [review]
